tests/macros/CSSxRef.test.js

```diff
@@ -177,7 +177,7 @@ const TEST_CASE = [
 // Test runner
 // ----------------------------------------------------------------------------
 
-describeMacro('cssxref', () => {
+describeMacro('CSSxRef', () => {
```

Comment:
I don't have an opinion about the capitalization of macro names, except that I don't think it is worth spending any effort to do it. But I do think that if the macro (and test) filenames are going to change it should be in a PR that is specific to that purpose, and not in this PR.